CHANGELOG.md

```diff
@@ -1,3 +1,36 @@
+# v0.12.0
+
+## Added
+
+- [[#187](https://github.com/rust-vmm/kvm-ioctls/pull/187)] Support for 
+  `KVM_SET_IDENTITY_MAP_ADDR`
+- Derive Debug for all exported structs and enums
+- [[#189](https://github.com/rust-vmm/kvm-ioctls/pull/189)] Expose `KVM_SET_` and 
+  `KVM_HAS_DEVICE_ATTR` for vcpu
+- [[#191](https://github.com/rust-vmm/kvm-ioctls/pull/191)] Add `KVM_TRANSLATE` and
+  `translate_gva` support
+- [[#190](https://github.com/rust-vmm/kvm-ioctls/pull/190)] Enable usage of `sync_regs`
+- [[#198](https://github.com/rust-vmm/kvm-ioctls/pull/198)] Expose details from 
```

Comment:
1. nit: I think we might want to say "return details about `KVM_EXIT_FAIL_ENTRY` in vCPU run". It is not clear what this refers to otherwise.